Update `astro-sdk` docs (#1602)
astronomer/docs: learn/astro-python-sdk-etl.md

    astro-sdk-python
    ```
    
2. If you're using the Astro CLI locally, add the following variable to the `.env` file of your Astro project:

    ```text
    export AIRFLOW__CORE__ENABLE_XCOM_PICKLING=True
    ```

>     We recommend using `AIRFLOW__CORE__XCOM_BACKEND` for production environment along with `AIRFLOW__ASTRO_SDK__XCOM_STORAGE_CONN_ID` and `AIRFLOW__ASTRO_SDK__XCOM_STORAGE_URL`. This enables XCOM data to be saved to a S3 or GCS location. For example:

Reviewer: ```suggestion
 To allow XCOM data to be saved to a S3 or GCS location, Astronomer recommends using `AIRFLOW__CORE__XCOM_BACKEND`, `AIRFLOW__ASTRO_SDK__XCOM_STORAGE_CONN_ID`, and `AIRFLOW__ASTRO_SDK__XCOM_STORAGE_URL` in production environments. For example:
```

Reviewer 2: @manmeetkaur If users are using the latter option they don't need the `export AIRFLOW__CORE__ENABLE_XCOM_PICKLING=True` env variable. So it would be nice if we can make this explicit. :)

>     We recommend using `AIRFLOW__CORE__XCOM_BACKEND` for production environment along with `AIRFLOW__ASTRO_SDK__XCOM_STORAGE_CONN_ID` and `AIRFLOW__ASTRO_SDK__XCOM_STORAGE_URL`. This enables XCOM data to be saved to a S3 or GCS location. For example:

Reviewer 2: @manmeetkaur When we use the `AIRFLOW__CORE__XCOM_BACKEND` option we no longer need the `AIRFLOW__CORE__ENABLE_XCOM_PICKLING=True` it would be nice if we can make this more explicit. :)

    ```text
    export AIRFLOW__CORE__XCOM_BACKEND='astro.custom_backend.astro_custom_backend.AstroCustomXcomBackend'
    export AIRFLOW__ASTRO_SDK__XCOM_STORAGE_CONN_ID=<my_aws_conn>
    export AIRFLOW__ASTRO_SDK__XCOM_STORAGE_URL='s3://my-bucket/xcom/'
    ```
    

>     If you would like to store all the intermediary tables generated by `astro-sdk` in a schema of your choice, you can specify the schema by using variable `AIRFLOW__ASTRO_SDK__SQL_SCHEMA`. To deploy a pipeline written with the Astro Python SDK to Astro, add these environment variables to your Deployment. See [Environment variables](https://docs.astronomer.io/astro/environment-variables).

Reviewer: ```suggestion
    To store all of the intermediary tables generated by `astro-sdk` in a specific schema, use the variable `AIRFLOW__ASTRO_SDK__SQL_SCHEMA` to specify the schema. To deploy a pipeline written with the Astro Python SDK to Astro, add these environment variables to your Deployment. See [Environment variables](https://docs.astronomer.io/astro/environment-variables).
```
